src/lib/pages/podcast-gallery/lib/load-podcasts.ts

```diff
@@ -0,0 +1,15 @@
+import type { Load } from '@sveltejs/kit';
+import { podcastList, podcasts } from '$lib/entities/podcast';
+
+export const loadPodcasts: Load = async () => {
```

Comment:
Question: shouldn't this file be in the `api` slice instead of `lib`?

Reply:
Good call, I think it should